SimonImanigirimpuhwe/my-brand-website
# #10 ft(article): add form to create new article

#### What does this PR do?

- Enable Blog owner to create new blog post

#### Description of Task to be completed?

- Make a form for input changes

- Add scripts to display add article form

#### How should this be manually tested?

- login in to get redirected to the dashboard, you will see a plus sign for adding new posts

- Click on `plus icon` you should be able to see the form displaying in the dashboard


#### Any background context you want to provide?

- The admin dashboard page is created at the time of registration

#### What are the relevant Trello stories?

- [#10](https://trello.com/c/WG6ttrzn/10-create-article-page)

#### Screenshots (if appropriate)

![add-article-screenshoot](https://user-images.githubusercontent.com/54619678/93815182-dbe72600-fc55-11ea-8c4a-326c2f0b4558.png)


#### Questions:

- N/A
